.travis.yml

```diff
@@ -0,0 +1,14 @@
+dist: trusty
+
+os: linux
+
+install: true
+
+cache:
+  directories: [ $HOME/.m2 ]
+
+language: java
+
+script:
+  - mvn install -DskipTests=true -Drat.skip=true -Dmaven.javadoc.skip=true -B -V
```

Comment:
As with apache/incubator-guacamole-server#110, I'm not confident Travis CI is a good choice. Depending on the result of discussion, if we do choose to go with Travis, skipping parts of the build shouldn't be done, especially tests and the RAT check. Catching test failures is one of the key points of using CI.

Reply:
as for skipping RAT check, I left that to those who understands how to deal with that :) I do not insist in skipping actually